runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -3418,6 +3418,9 @@ remoteCompile(J9VMThread *vmThread, TR::Compilation *compiler, TR_ResolvedMethod
          JITServer::ClientStream::incrementIncompatibilityCount(OMRPORT_FROM_J9PORT(compInfoPT->getJitConfig()->javaVM->portLibrary));
          }
 
+      if (TR::Options::getVerboseOption(TR_VerboseJITServerConns))
```

Comment:
You'll have to verify, but I think that the message I was talking about comes from this code in CompilationThread.cpp
```
   catch (const JITServer::StreamVersionIncompatible &e)
      {
      if (TR::Options::getVerboseOption(TR_VerboseJITServer))
         TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "compThreadID=%d JITServer StreamVersionIncompatible: %s", getCompThreadId(), e.what());
      Trc_JITServerStreamVersionIncompatible(vmThread, getCompThreadId(),  __FUNCTION__, compiler->signature(), compiler->getHotnessName(), e.what());
      _methodBeingCompiled->_compErrCode = compilationStreamVersionIncompatible;
      }
```
I was thinking of enabling that message, rather than adding another one.